### PR TITLE
Updating observability disable metrics-collector 2.4

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/19785

### CODE REVIEW

File: observability/customize_observability.adoc [hunk 1]
```diff
@@ -269,9 +269,9 @@ You can disable the `metrics-collector`, which stops it from collecting the data
 [#disable-metrics-collector-on-all-clusters]
 === Disabling _metrics-collector_ on all clusters
```
Comment (marcolan018) [inline observability/customize_observability.adoc]: for the title === Disabling _metrics-collector_ on all clusters
I think
=== Disabling observability on all clusters
is better?

File: observability/customize_observability.adoc [hunk 2]
```diff
@@ -269,9 +269,9 @@ You can disable the `metrics-collector`, which stops it from collecting the data
...
 
-Disable the `metrics-collector` pod to stop data from being collected and sent to the observability service on the {product-title-short} hub cluster. 
+Disable the `metrics-collector` by removing observability components on all managed clusters. This stops data from being collected and sent to the observability service on the {product-title-short} hub cluster.
```
Comment (marcolan018) [inline observability/customize_observability.adoc]: now it's not disable the "metrics-collector" pod, but disable observability.
so maybe 
Disable observability by removing observability components on all managed clusters.
is better?

Comment (dockerymick) [reply observability/customize_observability.adoc]: @marcolan018 Is this only applicable to 2.4? Or should this be updated in all of the versions? 

Here are my other PRS:

https://github.com/stolostron/rhacm-docs/pull/3141/files#diff-3f412710648058338c234848ca6246fbed469f9082cee8e1719814eddaf872ac

https://github.com/stolostron/rhacm-docs/pull/3142/files#diff-61d8562a58617059ec4a709de398f7a6c707833c35ab20831dfd973f06ee0471

Comment (marcolan018) [reply observability/customize_observability.adoc]: @dockerymick this should apply to all PRs

File: observability/customize_observability.adoc [hunk 3]
```diff
@@ -282,13 +282,13 @@ spec:
...
-=== Disabling _metrics-collector_ on a single cluster
+[#disable-observability-on-a-single-cluster]
+=== Disable observability on a single cluster
 
-Disable the `metrics-collector` by removing observability components on specific managed clusters. Add the `observability: disabled` label to the `managedclusters.cluster.open-cluster-management.io` custom resource.
+Disable observability by removing observability components on specific managed clusters. Add the `observability: disabled` label to the `managedclusters.cluster.open-cluster-management.io` custom resource.
 
 From the {product-title-short} console _Clusters_ page, add the `observability=disabled` label to the specified cluster.
-
+//is this note accurate
```
Comment (dockerymick) [inline observability/customize_observability.adoc]: @marcolan018 is this accurate?

Comment (marcolan018) [reply observability/customize_observability.adoc]: yes, the label operation is still as same as before

**Pull request merged**